RNKeyboardEventsManager.m

```diff
@@ -73,6 +73,11 @@ - (NSDictionary*) makeBodyFromNotification:(NSNotification*)notification {
   CGRect keyboardEndRect = [[[notification userInfo] objectForKey:UIKeyboardFrameEndUserInfoKey] CGRectValue];
   NSNumber *durationValue = [[notification userInfo] objectForKey:UIKeyboardAnimationDurationUserInfoKey];
 
+    
+  // Normalization: if durationValue is null, set it to 0
+  if(durationValue == nil)
+    durationValue = [[NSNumber alloc] initWithDouble:0];
+
```

Comment:
Could you update the code to use curly braces please and use the convenience method, like this:

```
if (durationValue == nil) {
  durationValue = [NSNumber numberWithDouble:0];
}
```